android/electrode-reactnative-bridge/src/main/java/com/walmartlabs/electrode/reactnative/bridge/RequestRegistrarImpl.java

```diff
@@ -55,7 +64,10 @@ public boolean registerRequestHandler(@NonNull String name, @NonNull T requestHa
     public T unregisterRequestHandler(@NonNull UUID requestHandlerUuid) {
         String requestName = mRequestNameByUUID.remove(requestHandlerUuid);
         if (requestName != null) {
+            Logger.d(TAG, "Request handler(id: %s) removed for request: %s", requestHandlerUuid, requestName);
             return mRequestHandlerByRequestName.remove(requestName);
+        } else {
+            Logger.d(TAG, "Request handler(id: %s) already removed", requestHandlerUuid);
```

Comment:
The other two added `Logger` usages have info level, and here it's debug (with a very similar message) - Keep it consistent? Either debug or info would be fine I think..

Reply:
I will drop it to debug